addons/zeus/functions/fnc_moduleSetEngineer.sqf

```diff
@@ -0,0 +1,26 @@
+/*
+ * Author: mharis001
+ * Zeus module function to set unit engineer skill
+ *
+ * Arguments:
+ * 0: Unit <OBJECT>
+ * 1: Skill level (1 - Engineer, 2 - Adv. Engineer) <NUMBER>
+ *
+ * Return Value:
+ * None
+ *
+ * Example:
+ * [unit, 2] call ace_zeus_fnc_moduleSetEngineer
+ *
+ * Public: No
+ */
+
+#include "script_component.hpp"
+
+params [["_unit", objNull], ["_engineerN", 1]];
```

Comment:
Is there a case where `_unit` is not passed from ACE3 code? No need to use default value then, rather show an error so we can fix it at source.